Epsilon/Export/Exporters/WordModuleExporter.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.Drawing;
+using System.Text;
+using System.Text.RegularExpressions;
+using Epsilon.Abstractions.Export;
+using Epsilon.Canvas.Abstractions.Model;
+using ExcelLibrary.SpreadSheet;
+using Microsoft.Extensions.Options;
+using Xceed.Document.NET;
+using Xceed.Words.NET;
+using Alignment = Xceed.Document.NET.Alignment;
+
+namespace Epsilon.Export.Exporters;
+
+public class WordModuleExporter : ICanvasModuleExporter
+{
+    private readonly ExportOptions _options;
+
+    public WordModuleExporter(IOptions<ExportOptions> options)
+    {
+        _options = options.Value;
+    }
+
+    public IEnumerable<string> Formats { get; } = new[] { "word" };
+
+    public void Export(IEnumerable<Module> modules, string format)
+    {
+        using (DocX document = DocX.Create(_options.FormattedOutputName+".docx"))
+        { 
+            document.AddFooters();
+            var link = document.AddHyperlink("Epsilon", new Uri("https://github.com/Typiqally/epsilon"));
+
+            document.Footers.Odd
+                .InsertParagraph("Created with ")
+                .AppendHyperlink(link).Color( Color.Blue ).UnderlineStyle( UnderlineStyle.singleLine );
+                
+            foreach (var module in modules.Where(static m => m.Collection.OutcomeResults.Any())) 
+            {
+
+                var links = module.Collection.Links;
+                var alignments = links.AlignmentsDictionary;
+                var outcomes = links.OutcomesDictionary;
+                
+                var table = document.AddTable( 1, 3);
+
+                table.Rows[0].Cells[0].Paragraphs[0].Append("KPI");
+                table.Rows[0].Cells[1].Paragraphs[0].Append("Assignment(s)");
+                table.Rows[0].Cells[2].Paragraphs[0].Append("Score");
+                
+                foreach (var (outcomeId, outcome) in outcomes)
+                {
+                    var assignmentIds = module.Collection.OutcomeResults
+                        .Where(o => o.Link.Outcome == outcomeId)
+                        .Select(static o => o.Link.Assignment)
+                        .ToArray();
+
+                    if (assignmentIds.Any())
+                    {
+                        var row = table.InsertRow();
+                        row.Cells[ 0 ].Paragraphs[ 0 ].Append( outcome.Title + " " +  ShortDescription(ConvertHtmlToRaw(outcome.Description)) );
```

Comment:
Can you reformat the spaces around the array index and parenthesis? They do not conform to the coding style Also for the rest of this class.